src/main/java/it/sweven/blockcovid/routers/AdminRouter.java

```diff
@@ -83,7 +83,7 @@
       throw new ResponseStatusException(HttpStatus.FORBIDDEN, "User has not enough privileges");
   }
 
-  @PostMapping(
+  @PutMapping(
       value = "/user/{username}/modify",
```

Comment:
maybe we can change the link of the mapping to `/user/{username}` since the `PUT` request is a _modify_ contstraint per-se, but I'm open to discussion on this point

Reply:
It makes sense, but with the same logic the user creation should be remapped to a `POST` request to `/user/{username}`, and the user deletion to a `DELETE` request to the same URL.

Reply:
ok I get your point